### PR TITLE
[13.0][ADD] product_variant_attribute_name_manager

### PR DESCRIPTION
**Pull request by OriolVForgeFlow**
New module: product_variant_attribute_name_manager
Summary: Manage how to display the attributes on the product variant name.
@forgeflow

### CODE REVIEW

File: product_variant_attribute_name_manager/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "summary": """Manage how to display the attributes on the product variant name.""",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "category": "Product",
+    "version": "13.0.1.0.0",
+    "depends": ["product"],
+    "data": ["views/product_variant_attribute_name_manager_view.xml"],
+    "license": "AGPL-3",
+    "auto_install": False,
+    "installable": True,
```
Comment (JordiBForgeFlow) [inline product_variant_attribute_name_manager/__manifest__.py:12]: Please add yourself as maintainer: "maintainers": ["oriolvforgeflow"]

File: product_variant_attribute_name_manager/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Oriol Villamayor <oriol.villamayor@forgeflow.com> (www.forgeflow.com)
```
Comment (JordiBForgeFlow) [inline product_variant_attribute_name_manager/readme/CONTRIBUTORS.rst]: Remove the www.forgeflow.com part.

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Functional and technical review. Minor comments.

**Review by mariadforgeflow — APPROVED** [review]
LGTM! Functional and code review.

**Pull request merged**